mac

```diff
@@ -151,11 +150,27 @@ cask "docker"
 brew "postgres"
 brew "redis"
 
-# Basics
-cask "google-chrome"
-cask "slack"
 EOF
 
+# Basics Apps
+
+if [ ! -d "/Applications/Google Chrome.app" ]; then
+  fancy_echo "Installing Google Chrome.."
+  brew cask install google-chrome
+  fancy_echo "Google Chrome installed."
+fi
+
+if [ ! -d "/Applications/Slack.app" ]; then
+  fancy_echo "Installing Slack.."
+  brew cask install slack
+  fancy_echo "Slack installed."
+fi
+
+brew link --overwrite gnupg
```

Comment:
@fikrikarim why is this required?

Reply:
The `brew link --overwrite gnupg`?

### 4. gpg2 brew link error:
After installing brew gpg2, the following errors come up:
```bash
Error: The `brew link` step did not complete successfully
The formula built, but is not symlinked into /usr/local
Could not symlink bin/gpg
Target /usr/local/bin/gpg
already exists. You may want to remove it:
  rm '/usr/local/bin/gpg'


To force the link and overwrite all conflicting files:
  brew link --overwrite gnupg

To list all files that would be deleted:
  brew link --overwrite --dry-run gnupg
```
So the `brew link --overwrite gnupg` is added to the script file to solve the issue

[4.gpg2failed.log](https://github.com/nimbl3/laptop/files/2067546/4.gpg2failed.log)

Based on this [stackoverflow answer](https://apple.stackexchange.com/questions/281008/install-gpg-with-homebrew) I think it's safe to run them.